nba_api/stats/library/data.py

```diff
@@ -2333,15 +2369,15 @@
     [77346, "Lavelli", "Tony", "Tony Lavelli", False],
```

Comment:
```
[1630237, "Lamb", "Anthony", "Anthony Lamb", True]
```

Don't think I can comment on the specific line, but Anthony's a new addition to the list for the Rockets.